PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -2900,72 +2902,81 @@ void ScintillaEditView::changeCase(__inout wchar_t * const strWToConvert, const
 	} //switch (caseToConvert)
 }
 
-void ScintillaEditView::convertSelectedTextTo(const TextCase & caseToConvert)
-{
-	unsigned int codepage = _codepage;
-	UniMode um = getCurrentBuffer()->getUnicodeMode();
-	if (um != uni8Bit)
-	codepage = CP_UTF8;
+} // namespace
 
-	if (execute(SCI_GETSELECTIONS) > 1) // Multi-Selection || Column mode
-	{
-        execute(SCI_BEGINUNDOACTION);
+// Case converts the document byte range [start:end] in place and returns the
+// change in its length in bytes. On any error, does nothing and returns zero.
+intptr_t ScintillaEditView::caseConvertRange(intptr_t start, intptr_t end, TextCase caseToConvert)

```

Comment:
Please move this method at the end.

Reply:
Moving it after convertSelectedTextTo didn't fix the diff, but moving it before changeCase did. Sorry for the trouble.